### PR TITLE
Added ukho_tides

### PR DESCRIPTION
**Pull request by DrJohnT**
Added ukho_tides wheels custom integration file

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @DrJohnT, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#363).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: components/ukho_tides.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "ukho_tides",
+    "owner": ["ianByrne"],
+    "manifest": "https://raw.githubusercontent.com/ianByrne/HASS-ukho_tides/main/custom_components/ukho_tides/manifest.json",
+    "url": "https://github.com/ianByrne/HASS_Integration_UKHOTides"
+}
+
```
Comment (frenck) [inline components/ukho_tides.json:7]:
```suggestion
{
    "name": "ukho_tides",
    "owner": ["@ianByrne"],
    "manifest": "https://raw.githubusercontent.com/ianByrne/HASS-ukho_tides/main/custom_components/ukho_tides/manifest.json",
    "url": "https://github.com/ianByrne/HASS_Integration_UKHOTides"
}
```

**Comment by frenck** [comment]
@DrJohnT Is @ianByrne aware you are adding his integration to this repository?

**Comment by DrJohnT** [comment]
> @DrJohnT Is @ianByrne aware you are adding his integration to this repository?

Yes, I am helping him out on documentation and testing

**Review by frenck — APPROVED** [review]
Thanks, @DrJohnT 👍

**Pull request merged**